Consolidate and iterate performance reporting

benilovj opened the pull request:
## Context

The current performance dashboard makes it hard to see what's happening post-submission (how many successful and unsuccessful candidates there are) at a glance.

## Changes proposed in this pull request

Iterate the service performance dashboard:

- consolidate candidate and application form reporting into just candidate reporting, because it's effectively the same thing until Apply 2 is added
- add more focus to how many applications are in pre- and post-submission.
- make it clearer how many applications are in important end-states (offers accepted or unsuccessful)

Before:

![image](https://user-images.githubusercontent.com/23801/74428803-9e195800-4e51-11ea-957c-f7151ff84dee.png)

After:

![image](https://user-images.githubusercontent.com/23801/74521492-6245c780-4f11-11ea-8bc6-c1b2c3ce5154.png)

## Guidance to review

- is the visual design (layout and colours) OK?
- for the top-level metrics, is it clear what the number is from the labels?

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

duncanjbrown commented:
Great! I like very much seeing each layer as the sum of the layer below, which holds for the first two rows (183 = 54 +129). This isn't the case in the bottom-right.

Should we drop the "Dashboard" from the title? It seems a bit redundant

duncanjbrown commented:
What @tijmenb said 😂

fofr commented:
Maybe not orange for submitted. 🤔

benilovj commented:
Thanks all for the feedback – see the updated screenshot:

![image](https://user-images.githubusercontent.com/23801/74521492-6245c780-4f11-11ea-8bc6-c1b2c3ce5154.png)

Changes:

- I've pared down the colours, they can be iterated by and actual designer on a later PR
- I've updated the 3rd row of metrics so it adds up to the 2nd row, and the fonts are smaller
- I've removed the word 'dashboard' from the title

fofr commented on app/views/support_interface/performance/index.html.erb:
```diff
@@ -1,50 +1,78 @@
-<% content_for :title, 'Service performance dashboard' %>
+<% content_for :title, 'Service performance' %>
```
:+1:
